.travis.yml

```diff
@@ -29,6 +29,7 @@ install:
   source terryfy/travis_tools.sh; get_python_environment  $pydist $pyver; fi
 - pip install -U pip wheel setuptools
 - pip install -r requirements.txt
+- if [ $pyver != '2.7' ]; then pip uninstall -y futures; fi
```

Comment:
What @pprett suggested is that we split `requirements.txt` into 2 files:
- requirements-base.txt (the same as `requirements.txt` but w/o `futures`)
- requirements-python2.txt (with `futures`)
I am going to do that unless someone has better ideas.